extensions/shared/components/stripe-nudge/index.jsx

```diff
@@ -39,6 +39,7 @@ export default ( { blockName, postId, stripeConnectUrl } ) => {
 
 	return (
 		<BlockNudge
+			className="jetpack-stripe-nudge__banner"
```

Comment:
I guess this needs now a change in the `BlockNudge` component so it accepts an optional `className` property:

https://github.com/Automattic/jetpack/blob/aaacec603ca3390d45ee4b1f94accc0db7b66911/extensions/shared/components/block-nudge/index.jsx#L11